### PR TITLE
增加文章阅读次数

### PR DESCRIPTION
**Pull request by wangbin8909**
增加文章阅读次数

### CODE REVIEW

**Review by roberChen — APPROVED** [review]
approved, waiting for @gsw945

File: models/DocumentModel.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ type Document struct {
 	Version    int64     `orm:"column(version);type(bigint);" json:"version"`
 	//是否展开子目录：0 否/1 是 /2 空间节点，单击时展开下一级
 	IsOpen     int           `orm:"column(is_open);type(int);default(0)" json:"is_open"`
+	ViewCount  int       `orm:"column(view_count);type(int)" json:"view_count"`
```
Comment (gsw945) [inline models/DocumentModel.go:44]: please format this line with same Align with sibling lines

**Review by gsw945 — CHANGES_REQUESTED** [review]
please adjust code format

**Review by gsw945 — APPROVED** [review]

**Comment by gsw945** [comment]
[roberChen](https://github.com/roberChen) has approved, and new change is just formatting code, so merge it

**Pull request merged**